etc/config/c.amazon.properties

```diff
@@ -290,19 +290,80 @@ compiler.armv8-full-cclang-trunk.options=--gcc-toolchain=/usr/lib/gcc/x86_64-lin
 compiler.armv8-full-cclang-trunk.alias=armv8.5-cclang-trunk
 
 # Clang for RISC-V
-group.rvcclang.compilers=rv32cclang:rv64cclang
+group.rvcclang.compilers=rv32-cclang:rv32-cclang1100:rv32-cclang1001:rv32-cclang1000:rv32-cclang-900:rv64-cclang:rv64-cclang1100:rv64-cclang1001:rv64-cclang1000:rv64-cclang900
 group.rvcclang.groupName=Clang RISC-V
 group.rvcclang.supportsBinary=false
-compiler.rv32cclang.exe=/opt/compiler-explorer/clang-trunk/bin/clang
-compiler.rv32cclang.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
-compiler.rv32cclang.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.rv32cclang.name=RISC-V rv32gc clang (trunk)
-compiler.rv32cclang.options=-target riscv32 -march=rv32gc -mabi=ilp32
-compiler.rv64cclang.exe=/opt/compiler-explorer/clang-trunk/bin/clang
-compiler.rv64cclang.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
-compiler.rv64cclang.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
-compiler.rv64cclang.name=RISC-V rv64gc clang (trunk)
-compiler.rv64cclang.options=-target riscv64 -march=rv64gc -mabi=lp64
+
+compiler.rv32-cclang.name=RISC-V rv32gc clang (trunk)
```

Comment:
should include alias to the old id: rv32cclang
so compiler.rv32-cclang.alias=rv32cclang

Same with the 64bit of trunk

Reply:
Fixed now, I didn't see we could alias, and this is nicer for users :)